pkg/noderesourcetopology/filter.go

```diff
@@ -98,8 +99,9 @@ func resourcesAvailableInAnyNUMANodes(logKey string, numaNodes NUMANodeList, res
 		}
 		bitmask.And(resourceBitmask)
 		if bitmask.IsEmpty() {
-			klog.V(5).InfoS("early verdict", "logKey", logKey, "node", nodeName, "resource", resource, "suitable", "false")
-			return false
+			klog.V(5).InfoS("short circuit", "logKey", logKey, "node", nodeName, "resource", resource)
+			bitmask.Clear()
```

Comment:
Bitmask is already empty here, why do we need to clear it?

Reply:
sure, but I'm dropping this commit anyway